tests/test_user_panel.py

```diff
@@ -70,14 +73,32 @@ def create_order(assopy_user, fare):
     return order
 
 
-@pytest.mark.xfail
-def test_privacy_settings_requires_login():
-    assert False
+def test_privacy_settings_requires_login(client):
+    url = reverse('user_panel:privacy_settings')
 
+    response = client.get(url)
 
-@pytest.mark.xfail
-def test_privacy_settings_updates_profile():
-    assert False
+    assert redirects_to(response, reverse("accounts:login"))
+
+
+def test_privacy_settings_updates_profile(user_client):
+    url = reverse('user_panel:privacy_settings')
+    profile = user_client.user.attendeeprofile.p3_profile
+    assert not profile.spam_recruiting
```

Comment:
coming back to the `not` vs `is False` thing – it's technically speaking not just a matter of style.
`not` tests for anything that's Falsy, which (among zeros, empty lists, dicts) etc. includes also `None`.
Having `is False` is more explicit.

Reply:
I see your point - I think it makes sense to make it more explicit in cases like this one, where it's a model property, where the property might not be boolean. There's no need to do that for calls to `not queryset.exists()` since those do return bools. I'll update these tests.

Reply:
Had to refresh it - checking `not value` and `value is False` is equivalent. To test strict equivalence, I'll update it to `value == False`